src/install/lockfile.zig

```diff
@@ -3069,6 +3073,37 @@ pub const Package = extern struct {
             }
         }
 
+        if (comptime features.privileged_dependencies) {
+            if (json.asProperty("privilegedDependencies")) |q| {
+                switch (q.expr.data) {
+                    .e_array => |arr| {
+                        try lockfile.privileged_dependencies.ensureUnusedCapacity(allocator, arr.items.len);
+                        for (arr.slice()) |item| {
```

Comment:
i'm sorry this is a really annoying suggestion but

could we count the total length here and then use StringBuilder? These might become a long list

Reply:
Dropped strings altogether &minus; store and compares hashes instead 😉